### ISSUE TITLE
faster checksum phase

### ISSUE THREAD

**Issue by morgo** [cashapp/spirit#194]
Usually the checksum phase takes about 10-20% of the time that the copier phase does. However, on some very IO bound cases I have noticed it take significantly longer.

Much earlier in development we offered a separate `checksum-threads` concurrency setting, but unified it with threads to simplify configuration.

Because in resume-from-checkpoint we resume from the start of the checksum again, it's important that we keep the checksum fast. We could either use a static rule like double the threads used on copier + double the chunk time, or we could offer a configuration variable. TBD.

**Comment by morgo** [comment]
It's difficult to implement this; in some cases checksum can incur significant load because the copier is effectively throttled by waiting on IO writes, but the checksum may not be.

I am going to close this as won't fix.
